bip-truc.mediawiki

```diff
@@ -0,0 +1,271 @@
+<pre>
+  BIP: ???
+  Layer: Applications
+  Title: Topologically Restricted Until Confirmation Transactions
+  Author: Gloria Zhao <gloriajzhao@gmail.com>
+  Comments-URI:
+  Status: Draft
+  Type: ???
+  Created: 2024-01-10
+  License: BSD-3-Clause
```

Comment:
Please add the "Post-History" header in the preamble and link to relevant mailing list threads and other discussions (e.g. delving).

Reply:
done, thanks